woocommerce/woocommerce-android
# Issue/6586 wcs onboarding cta screen

Part of: #6586 


### Description
Displays new onboarding screen before triggering the WC Shipping installation flow. Designs for this screen are here 9PGX5zb3VKHMYc7qTIv30B-fi-744%3A24627

![Screenshot 2022-06-03 at 12 09 53](https://user-images.githubusercontent.com/2663464/171834833-667e300e-a405-4e99-aae8-85eb0667a199.png)


### Testing instructions
To access this new screen a site eligible for shipping labels is required for the [new banner](https://github.com/woocommerce/woocommerce-android/pull/6620) to be displayed inside OrderDetail screen. Conditions for the banner to be shown:

- The store is located in US.
- The order currency is USD.
- The products are eligible for SL (not virtual/downloadable).
- The order is not eligible for IPP.
- WC Shipping plugin must be uninstalled or unactive

Once the banner is shown, click on "Get WooCommerce Shipping" button and the new screen should be displayed:


https://user-images.githubusercontent.com/2663464/171835793-0e9a015e-47a9-49ba-95f1-f4af03d5fba6.mp4
